hera_mc/tests/test_correlator.py

```diff
@@ -1464,9 +1464,7 @@ def test_snap_status_errors(mcsession):
 
 
 def test_get_node_snap_from_serial_nodossier(mcsession):
-    node, snap_loc_num = checkWarnings(mcsession._get_node_snap_from_serial,
-                                       ['foo'],
-                                       message='No active dossiers')
+    node, snap_loc_num = checkWarnings(mcsession._get_node_snap_from_serial, ['foo'])
```

Comment:
This is going backwards, you need to check that the message you get is the correct message.

Reply:
Added text check.